src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -295,13 +295,15 @@ def getter():
         return _generic_site_operation(cmd.cli_ctx, resource_group_name, name, 'get', slot)
 
     def setter(webapp):
-        webapp.identity = ManagedServiceIdentity(type='SystemAssigned')
+        if disable_msi:
```

Comment:
The right thing is to implement the `az webapp identity remove` based on that the `none` is working now

Reply:
ok so should we remove the --disable-msi property from assign then & just implement the new az webapp identity remove?

Reply:
correct

Reply:
@mattchenderson FYI - hope this plan sounds good to you as well. Please let us know  otherwise.